WcaOnRails/app/views/competitions/_my_competitions_table.html.erb

```diff
@@ -0,0 +1,54 @@
+<% if competitions.length < 1 %>
+  <% message = upcoming ? "You currently have no upcoming competitions! Check out the #{link_to 'competitions list', '/results/competitions.php'}.".html_safe : "You have no past competitions." %>
+  <div class="alert alert-info" role="alert">
+    <%= message %>
+  </div>
+<% else %>
+  <table class="table competitions-table floatThread <%= !upcoming ? "table-striped" : "" %>">
+    <tr>
+      <th>Name</th>
+      <th>Location</th>
+      <th>Date</th>
+      <th></th>
+      <th></th>
+      <th class="big-column"></th>
+    </tr>
+    <% competitions.each do |competition| %>
+      <% if competition.isConfirmed %>
+        <% if competition.showAtAll %>
+          <% message = "This competition is confirmed and visible." %>
+        <% else %>
+          <% message = "This competition is confirmed but not visible." %>
+        <% end %>
+      <% else %>
+        <% if competition.showAtAll %>
+          <% message = "This competition is not confirmed but is visible." %>
+        <% else %>
+          <% message = "This competition is not confirmed and not visible." %>
+        <% end %>
+      <% end %>
+      <% if upcoming %>
+        <tr class="<%= competition.isConfirmed ? "confirmed" : "not-confirmed" %> <%= competition.showAtAll ? "visible" : "not-visible" %>" data-toggle="tooltip" data-placement="top" data-container="body" title="<%= message %>">
+      <% else %>
+        <tr>
+      <% end %>
+        <td><%= link_to competition.name, competition_path(competition) %></td>
+        <td><%= competition.cityName %>, <%= competition.country_name %></td>
+        <td><%= date_range(competition.start_date, competition.end_date, separator: "-") %></td>
+        <td>
+          <% if upcoming && current_user.can_manage_competition?(competition) %>
+            <%= link_to "Edit", edit_competition_path(competition) %>
+          <% elsif competition.results.length > 0 %>
+            <span class="glyphicon glyphicon-ok-sign" aria-hidden="true" data-toggle="tooltip" data-placement="top" data-container="body" title="Results are up!"></span>
+          <% end %>
+        </td>
+        <td>
+          <% if upcoming && competition.use_wca_registration? && current_user.can_manage_competition?(competition) %>
+            <%= link_to "Registrations", competition_edit_registrations_path(competition) %>
+          <% end %>
+        </td>
+        <td>
```

Comment:
Looks like a `<td>` without a closing `</td>`? Is this td for the `big-column`? If so, I think it would make things clearer if you gave it the class `big-column`.